.style-and-markdown-build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+ret=0
+
+if grep -rn $'\t' modules assemblies pom.xml --include=pom.xml; then
+  echo "Tabs found!"
+  ret=1
+fi
+if grep -rn ' $' modules assemblies pom.xml --include=pom.xml; then
+  echo "Trailing spaces found!"
+  ret=1
+fi
+if grep -rn '	' etc; then
+  echo "Tabs found in config files!"
+  ret=1
+fi
+if grep -rn ' $' etc; then
+  echo "Trailing spaces found in config files!"
+  ret=1
+fi
+
+
+# build number must be present in all modules
+if ! grep -L '<Build-Number>${buildNumber}</Build-Number>' modules/*/pom.xml | wc -l | grep -q '^0$'; then
+  echo "Build number is missing from a module!"
+  ret=1
+fi
+
+# maven-dependency-plugin should be active for all new modules
+#grep -L maven-dependency-plugin modules/*/pom.xml > maven-dependency-plugin.list
+#if ! diff -q maven-dependency-plugin.list docs/checkstyle/maven-dependency-plugin.exceptions; then
+#  rev=1
+#fi
+
+cd docs/guides
+
+for docs in admin developer user; do
+  pushd . > /dev/null
+  cd $docs
+  mkdocs build > mkdocs.log 2>&1
+  if grep \
+       -e 'WARNING.*Documentation file' \
+       -e 'pages exist in the docs directory' \
+       -e 'is not found in the documentation files' \
+        mkdocs.log;
+    then
+      echo "$docs did not build correctly!"
+      ret=1
+  fi
+  echo "Markdown doc $docs build log:"
+  cat mkdocs.log
```

Comment:
Is it intentional that `mkdocs.log` is always printed? Before it was only printed in case there was an error. Both are fine, I just want to make sure it's intentional.

If you want to print it regardless, you can use `tee` above:
```sh
mkdocs build 2>&1 | tee mkdocs.log
```

Reply:
I thought it handy?  I ran into issues getting markdown to fail during testing, so I wanted to be sure the log contained *everything* and that my checks were at fault :)